sim_pipeline/image_simulation.py

```diff
@@ -62,3 +63,47 @@ def sharp_image(lens_class, band, mag_zero_point, delta_pix, num_pix, with_defle
                               kwargs_ps=kwargs_ps, unconvolved=True, source_add=True, lens_light_add=with_deflector,
                               point_source_add=False)
     return image
+
+
+def gsobj(image, pix_scale, flux):
```

Comment:
describe the function, also the name might not be obvious, perhaps re-defineit to galsimobj()?